cfme/utils/appliance/console.py

```diff
@@ -116,14 +116,14 @@ def scap_check_rules(self):
             if elements:
                 result = elements[0].findall('./{http://checklists.nist.gov/xccdf/1.1}result')
                 if result:
-                    if result[0].text != 'pass':
-                        rules_failures.append(rule)
+                    rules_failures.append(False) if result[0].text != 'pass' else (
```

Comment:
Looks like the functionality of the method is changing. @jaryn suggested to return a iterable list of failures (which is was doing), this is now just returning true or false. The doc block just needs to be updated.

Reply:
@jawatts  Updated doc string